### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by olimart**
## Description

- Fix typo `fo` --> `for`

## Release notes

Notes:

### CODE REVIEW

**Comment by Btrfly79** [comment]
I would just like everyone to know I did not write, copy, edit or publish any of the code that was on my account.  My personal account was hacked.  The closer I get to finding this guy (who I do t know btw) working with the FBI, local law enforcement, public, private abs federal agencies.  The closer I get the more he does things to hurt me.  I am an experienced and well educated Healthcare IT professional with a multitude of skills including developing a software program (after the code was written by It pros, I gave guidance on what we needed only).  I have never had hands on IT  coding experience.  The little I do k ie I learned by reading the code embedded on my fevives and honestly googling and asking professionals.  I am getting much better at reading code and to me when I say reading it  I mean enough to notice if it’s missing information or has too much or straight up makes no sense.  Do while I can appreciate all of you who are experienced seeing what looked like an inexperienced IT cider trying to get ahead, I was working with the people involved and my concerns were on the potential law suit I was facing.  Not the first on the last 5 months with this guy cyber stalking and harassing me.  But as I was getting back on GirHub to comment.  Which btw I did take a few screenshots of this individual’s code only because I knew it was him because I’m addition to it having many of the same elements on the code he’s embedded on my personal devices, the articles were all ones I had been reading so I can finish my report and other duties involved in this new project.  Well I assume it was then that he got into my GitHub account and upgraded to an enterprise account and must have added himself because now even though it’s cancelled, and nothing I wanted nor am I anywhere near ready for, I have to deal with another invasion on my privacy.   I do understand what it myst have looked like to you guys, but I would like to kindly ask that in the future you get more information before making hurtful comments.  You have no idea how much this guy has taken from me.  Snd I wanted to learn something new, or at least try it out.  No expectations and again he strikes.  There is way more to h the story.  I’d be happy to explain in detail what’s been going on from a personal perspective.  I can’t speak on other areas.  But I’d like you to have context, not just for me but for anyone I can te because if I can save one person from this it’ll be worth it.  And honestly before all this I was excited to jump in and had expectations of working with all of the professionals in this community and I’d still like that if anyone is willing.  If not, that’s ok too but I wanted to get my side heard.  I will take full accountability for my mistakes, but I refuse to let this guy take anything else away from me.  And regardless of what you think if me.  This Jew program is amazing and already helped so many and we are confident it will continue to get better.  For i digit, businesses, our nations’s economy and finding Covid research.  So while I hope we can move forward together, I may not if I can’t shake the hacker snd you might not want to.  Regardless, please don’t let any of this affect your opinion of the changes coming out.  Whether you like them or hate them, I hope you decide based on personal experience and not this incident.  I’m not confident in my email until I get my replacement devices so if anyone dies have questions or wants more details to give you a better understanding of this situation as a whole, feel free to call me at 808-225-6179.  My name is Christine Cruz.  Thank you for listening.

File: script/draft-release/run.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ function printInstructions(nextVersion: string, entries: Array<string>) {
 }
 
 /**
- * adds a number to the beginning fo each line and prints them in sequence
+ * adds a number to the beginning for each line and prints them in sequence
```
Comment (billygriffin) [inline script/draft-release/run.ts:92]:
```suggestion
 * adds a number to the beginning of each line and prints them in sequence
```

**Review by niik — APPROVED** [review]
Thank you for the contribution!

**Review by billygriffin — CHANGES_REQUESTED** [review]
I think this is close @olimart but I'm pretty sure it should be "of," not "for." Thanks!

**Comment by billygriffin** [comment]
Well looks like we already merged it, so it's probably fine 😆

File: script/draft-release/run.ts (hunk 1 above)
Comment (olimart) [reply script/draft-release/run.ts:92]: > but I'm pretty sure it should be "of," not "for." Thanks!

I'm pretty sure you're 💯 right. How did I miss that? 😄

Comment (niik) [reply script/draft-release/run.ts:92]: I'm sorry, I was a bit too trigger happy here. If you'd like to open up another PR @olimart we'd happily review that as well.

Comment (olimart) [reply script/draft-release/run.ts:92]: Done in #11065

**Pull request merged**